hack/verify-gofmt.sh

```diff
@@ -17,7 +17,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-diff=$(find . -name "*.go" | grep -v "\/vendor\/" | xargs gofmt -s -d 2>&1)
+cmd="bazel run //:gofmt -- -s -d"
+if ! which bazel &> /dev/null; then
+  echo "Bazel is the preferred way to build and test the test-infra repo." >&2
+  echo "Please install bazel at https://bazel.build/ (future commits may require it)" >&2
+  cmd="gofmt -s -d"
```

Comment:
same suuuper tiny nit: as with gofmt above, feel free to ignore

Reply:
Done